### PR TITLE
TRUNK-5040: Fix for inefficient use of map

### PR DESCRIPTION
**Pull request by meinsen**
## Issue I worked on

see https://issues.openmrs.org/browse/TRUNK-

## Checklist: I completed these to help reviewers :)

- [ ] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

  No? -> [read here](https://wiki.openmrs.org/display/docs/Pull+Request+Tips) on how to squash multiple commits into one

- [x] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [ ] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [x] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [x] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [x] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/ModuleFactory.java [hunk 1]
```diff
@@ -194,7 +194,8 @@ public static void loadModules(List<File> modulesToLoad) {
 					try {
 						Module mod = loadModule(f, true); // last module loaded wins
 						log.debug("Loaded module: " + mod + " successfully");
-					} catch (Exception e) {
+					}
```
Comment (teleivo) [inline api/src/main/java/org/openmrs/module/ModuleFactory.java]: this is not according to our style. see https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE

File: api/src/main/java/org/openmrs/util/databasechange/ConceptValidatorChangeSet.java [hunk 2]
```diff
@@ -243,7 +242,7 @@ else if (nameInLocale.getLocalePreferred() && preferredNameForLocaleFound) {
 					
 					nameDuplicateConceptNamesMap.get(name).add(nameInLocale);
 					
-				}//close for each name
+				} //close for each name
```
Comment (teleivo) [inline api/src/main/java/org/openmrs/util/databasechange/ConceptValidatorChangeSet.java]: please remove comment

File: api/src/main/java/org/openmrs/util/databasechange/ConceptValidatorChangeSet.java [hunk 3]
```diff
@@ -294,7 +278,7 @@ else if (nameInLocale.getLocalePreferred() && preferredNameForLocaleFound) {
 					}
 				}
 				
-			}//close for each locale
+			} //close for each locale
```
Comment (teleivo) [inline api/src/main/java/org/openmrs/util/databasechange/ConceptValidatorChangeSet.java]: please remove comment

File: api/src/main/java/org/openmrs/module/ModuleFactory.java (hunk 1 above)
Comment (meinsen) [inline api/src/main/java/org/openmrs/module/ModuleFactory.java]: do i understand right that you want to have closing brace, catch and opening brace on one line?
the code formatter changes that to 
closing brace on one line and 
catch and opening brace on the next line

Comment (meinsen) [inline api/src/main/java/org/openmrs/module/ModuleFactory.java]: @teleivo i created a new pull request as i messed up the git commit history with sqash to much and push to early. still learning git,
i don't see  where i can remove this pull request but will try further.

**Pull request closed without merging**